LibVLCSharp/Shared/Events/EventManager.cs

```diff
@@ -85,7 +116,4 @@ internal protected void OnEventUnhandled(object sender, EventType eventType)
         internal protected abstract void AttachEvent<T>(EventType eventType, EventHandler<T> eventHandler) where T : EventArgs;
```

Comment:
I'm guessing that with your refactoring `#if APPLE` above (can't select it in github) should not be needed anymore?